addons/point_of_sale/static/src/js/ChromeWidgets/ClientScreenButton.js

```diff
@@ -29,16 +30,19 @@ odoo.define('point_of_sale.ClientScreenButton', function(require) {
             }
         }
         async onClickLocal() {
-            this.env.pos.customer_display = window.open('', 'Customer Display', 'height=600,width=900');
+            // Before, customer_display was in 'pos', which makes it available "globally".
+            // This time however, we are putting it in a different global place since
+            // we can't attached dom element in 'pos'.
+            pos_env.customer_display = window.open('', 'Customer Display', 'height=600,width=900');
```

Comment:
Why do you have this pos_env stuff? In other components you have "this.env" that replaces "this.env.pos".

BTW, isn't it possible to keep "this.env.pos" to avoid useless diff in PR which is already very long?

Reply:
BTW "this.env.pos" seems to be available as it is already used elsewhere

Reply:
I'll remove this pos_env import in this component. But we need pos_env in the models as models are reactive and env will be become reactive when access thru pos. Making the env reactive is not a good idea (actually, pos won't work if we do so).